model/aro_hcp/v1alpha1/node_pool_resource.model

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific nodepool.
+resource NodePool {
+	// Retrieves the details of the node pool.
+	method Get {
+		out Body NodePool
+	}
+
+	// Updates the node pool.
+	method AsyncUpdate {
```

Comment:
I lack some context. What's the diff between Update and AsyncUpdate methods?

Reply:
The former returns `StatusOK` while the latter returns `StatusAccepted`

Reply:
{see previous comment by @ahitacat }